.eslintrc.js

```diff
@@ -6,7 +6,10 @@ const process = require('process');
 const isRelaxed = !!process.env.SENTRY_ESLINT_RELAXED;
 
 module.exports = {
-  extends: [isRelaxed ? 'sentry-app' : 'sentry-app/strict'],
+  extends: [
+    'plugin:react-hooks/recommended',
```

Comment:
What is in their recommended? mainly interested to know what is set to warn vs error

Reply:
Its just
"react-hooks/rules-of-hooks": "error",
"react-hooks/exhaustive-deps": "warn"